frontend/packages/client/src/api/notificationService.js

```diff
@@ -0,0 +1,117 @@
+import {
+  LP_APP_ID,
+  LP_CRO_KEY,
+  LP_EXPORT_KEY,
+} from 'api/constants';
+import Leanplum from 'leanplum-sdk';
+
+const COMMUNITY_UPDATES_CATEGORY_ID = 1;
+const options = {
+  method: 'GET',
+  headers: { accept: 'application/json' },
+};
+
+export const setUserId = async (walletId) => {
+  try {
+    Leanplum.setUserId(walletId);
+  } catch (e) {
+    throw new Error(e);
+  }
+};
+
+export const getUserSettings = async (walletId) => {
+  try {
+    fetch(
+      `https://api.leanplum.com/api?appId=${LP_APP_ID}&clientKey=${LP_EXPORT_KEY}&apiVersion=1.0.6&userId=${walletId}&action=exportUser`,
+      options
+    ).then((response) => {
+      let data = response.response[0];
```

Comment:
This looks a little odd to me, the Response constructor doesn't contain a response property according to the documentation, https://developer.mozilla.org/en-US/docs/Web/API/Response

Reply:
after you transform the response body stream to json then it takes on this object result array shape